test/powershell/Modules/Microsoft.PowerShell.Core/Import-Module.Tests.ps1

```diff
@@ -200,3 +200,34 @@ Describe "Import-Module should be case insensitive" -Tags 'CI' {
         Get-Module tESTmODULE | Should -BeNullOrEmpty
     }
 }
+
+Describe "Workflow .Xaml module is not supported in PSCore" -tags "CI" {
```

Comment:
It seems like this test doesn't need to be "CI".  We don't expect someone to re-implement workflows.  Maybe "Feature"?

Reply:
Sure, will make it a `Feature` test.